src/components/NewTabs/tabs.module.css

```diff
@@ -0,0 +1,27 @@
+.tabGroup p {
+    @apply text-sm;
```

Comment:
seems this this `p` should be styled in the productFullDetail component as it doesn't deppend on tab stylings